python/ray/serve/_private/deployment_state.py

```diff
@@ -1516,6 +1516,18 @@ def deploy(self, deployment_info: DeploymentInfo) -> bool:
                     deployment_info.target_capacity,
                     deployment_info.target_capacity_direction,
                 )
+
+            downscale_smoothing_factor = (
+                autoscaling_policy_manager.config.get_downscale_smoothing_factor()
+            )
+            if downscale_smoothing_factor <= 0.5:
+                logger.warning(
+                    "The downscale smoothing factor for deployment "
+                    f"'{self.deployment_name}' is set to a low value: "
+                    f"{downscale_smoothing_factor}. Note that this means the "
+                    "deployment is unlikely to scale below "
+                    f"{int(1 / downscale_smoothing_factor)} replicas."
```

Comment:
```suggestion
                    "The downscale smoothing factor for deployment "
                    f"'{self.deployment_name}' is set to a low value: "
                    f"{downscale_smoothing_factor}. This means the "
                    "deployment is unlikely to scale below "
                    f"{int(1 / downscale_smoothing_factor)} replicas."
```

Reply:
Can we also add a minimum recommended smoothing factor to the error message?

Reply:
Sure, I put the recommended min as 0.6. I think anything above 0.5 is an OK recommendation, but any number between 0.5-0.6 seems kind of arbitrary.